File: PushPopDemo/PushPopDemo/PushPopDemo/ViewController.m
//
//  ViewController.m
//  PushPopDemo
//
//  Created by chuckchan on 2020/4/15.
//  Copyright © 2020年 chuckchan. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


-(IBAction)done:(UIStoryboardSegue *)seque {
    [self.navigationController popViewControllerAnimated:YES];
}
@end

File: UITextView2/UITextView2/ViewController.swift
//
//  ViewController.swift
//  UITextView2
//
//  Created by chuck chan on 15/11/11.
//  Copyright © 2015年 chuck chan. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    @IBOutlet weak var chuckBtn: UIButton!

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    @IBAction func receiveAction(sender: UIButton) {
        self.resignFirstResponder();
    
    }
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

